src/containers/setup/user/ViewersContainer.js

```diff
@@ -0,0 +1,111 @@
+import React, { Component } from 'react';
+import Components from '../../../components';
+import { connect } from 'react-redux';
+import Validator from 'validator';
+
+import List, {
+  ListItem,
+  ListItemText,
+} from 'material-ui/List';
+
+import Dialog, {
+  DialogContent,
+  DialogTitle,
+} from 'material-ui/Dialog';
+
+import TextField from 'material-ui/TextField';
+import Button from 'material-ui/Button';
+
+import Actions from "../../../actions";
+
+const Panel = Components.Misc.Panel;
+
+class NewViewerForm extends Component {
+  state = { email: '' }
+
+  createViewer = () => this.props.createViewer(this.state.email)
+
+  render() {
+    const emailInvalid = !Validator.isEmail(this.state.email);
+    return (
+      <div>
+        <TextField
+          onChange={ event => this.setState({ email: event.target.value }) }
+          value={ this.state.email }
+          fullWidth
+          label='Email'
+          error={ emailInvalid }
+          helperText='Email for the user you will create and give read-only access to your data. The password will be generated later.'
+        />
+        <Button
+          onClick={ this.createViewer }
+          disabled={ emailInvalid }
+        >
+          Create
+        </Button>
+      </div>
+    )
+  }
+}
+
+class ViewersContainer extends Component {
+  state = { addViewerDialogOpen: false };
+
+  componentWillMount() {
+    console.log('viewers container props', this.props);
+    this.props.getViewers();
+  }
+
+  openDialog = addViewerDialogOpen => event => {
+    event.preventDefault();
+    this.setState({ addViewerDialogOpen });
+  }
+
+  render() {
+    console.log('viewer container props', this.props)
+    return (
+      <Panel>
```

Comment:
You display to white panels (one empty) on the page

Reply:
Seems to be related to your `Dialog` below. Check `AccountsContainer` to see how to fix that.